autospec/files.py

```diff
@@ -49,12 +49,31 @@ def __init__(self):
         # /usr/include?  Yes in the general case, but for example for R
         # packages, the answer is No.
         self.want_dev_split = True
+        self.has_banned = False
+
+    @staticmethod
+    def banned_path(path):
+        """Check if the path is either banned or in a banned subdirectory."""
+        banned_paths = ["/etc",
+                        "/opt",
+                        "/usr/local",
+                        "/usr/etc",
+                        "/usr/src"]
```

Comment:
Add `/var` to the list as well?

Reply:
Ah indeed, /var is something we run into now and then too thanks!